provider/vsphere/internal/vsphereclient/createvm.go

```diff
@@ -170,22 +189,22 @@ func (c *Client) ensureTemplateVM(
 	c.logger.Debugf("creating template VM in folder %s", vmFolder)
 	c.logger.Tracef("import spec: %s", pretty.Sprint(importSpec))
 
-	// Each controller maintains its own image cache. All compute
-	// provisioners (i.e. each model's) run on the same controller
-	// machine, so taking a machine lock ensures that only one
-	// process is updating VMDKs at the same time. We lock around
-	// access to the series directory.
-	mutexReleaser, err := mutex.Acquire(mutex.Spec{
-		Name:  "juju-vsphere-" + args.Series,
-		Clock: args.Clock,
-		Delay: time.Second,
-	})
-	if err != nil {
-		return nil, errors.Annotate(err, "acquiring lock")
+	if !args.IsBootstrap {
+		// Each controller maintains its own image cache. All compute
+		// provisioners (i.e. each model's) run on the same controller
+		// machine, so taking a machine lock ensures that only one
+		// process is updating VMDKs at the same time. We lock around
+		// access to the series directory.
+		release, err := acquireMutex(mutex.Spec{
+			Name:  "juju-vsphere-" + args.Series,
```

Comment:
we want to remove the 'juju-' prefix because it is automatically added by the mutex code.